docs/static/releasenotes.asciidoc

```diff
@@ -47,6 +48,109 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-11-0]]
+=== Logstash 8.11.0 Release Notes
+
+[[docs-8.11.0]]
+==== Documentation enhancements
+
+* Add native Logstash to Logstash documentation https://github.com/elastic/logstash/pull/15346[#15346]
+* Expanded description of how to size the JVM memory https://github.com/elastic/logstash/pull/15210[#15210]
+
+[[dependencies-8.11.0]]
+==== Updates to dependencies
+
+* Update Guava dependency to 32.1.2 https://github.com/elastic/logstash/pull/15394[#15394]
+* Downgrade jruby, keep updated default-gem dependencies https://github.com/elastic/logstash/pull/15369[#15369]
+* Pin psych 5.1.0 https://github.com/elastic/logstash/pull/15433[#15433]
+
+[[known-issues-8.11.0]]
+==== Known issues
+
+* Input imap plugin's behaviour with Logstash 8.10+ versions, is broken after upgrading its `mail` dependency https://github.com/logstash-plugins/logstash-input-imap/issues/61[#61]
```

Comment:
Have all known issues from 8.10.x been resolved in 8.11.0?

Reply:
Correct!